packages/compass-generative-ai/src/components/generative-ai-input.tsx

```diff
@@ -442,6 +442,7 @@ function GenerativeAIInput({
                 )}
                 ref={promptTextInputRef}
                 data-testid="ai-user-text-input"
+                aria-labelledby=""
```

Comment:
I think "deduping" react types uncovered some legit typescript issues that weren't showing up before for some reason, like this property is expected on leafygreen text area, but for some reason the typecheck wasn't catching it until now